### ISSUE TITLE
Error: TypeError: 'method' object is not subscriptable

### ISSUE THREAD

**Issue by johnsturgeon** [rsnodgrass/hass-sensorpush#25]
I'm getting:

```
Logger: homeassistant.util.logging
Source: util/logging.py:116 
First occurred: 8:23:42 AM (800 occurrences) 
Last logged: 9:42:42 AM

Exception in _update_callback when dispatching 'sensorpush_updated': () Traceback (most recent call last): File "/config/custom_components/sensorpush/__init__.py", line 149, in _update_callback sensor_results = samples['sensors'] TypeError: 'method' object is not subscriptable
```

And I'm not alone:
https://community.home-assistant.io/t/sensorpush-humidity-and-temperature-sensors/105711/53

**Comment by rsnodgrass** [comment]
New HA integration version released that handles a samples() API change in the underlying pysensorpush library. It should be working again. I do not have any SensorPush gateways/sensors anymore wth HA, so cannot test myself.

**Comment by johnsturgeon** [comment]
FYI, Tested and it works.  Also, I'm ditching SensorPush because of its reliance on the cloud.  Going to purchase 5 Aqara sensors instead.

Thank you for the fix though.  I'm sure other SensorPush users will appreciate it.

**Comment by rsnodgrass** [comment]
Agreed. Trying to not use any IoT devices which do not have a local API (or open source on device code so if cloud/company goes away) there is some ability to keep using I. The future. Have wasted thousands on IoT devices and sensors that no longer work.   
  

  
Would love to see more sensors on the Helium network with open data models so even if company goes away they work.   
  
  
  
  
  
  
  

  
  
>
